### PR TITLE
Shorten OIE upgrade tree nav titles

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** This PR shortens the tree nav titles for the "Okta Identity Engine upgrade" section of the guides.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-537858](https://oktainc.atlassian.net/browse/OKTA-537858)

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js [hunk 1]
```diff
@@ -404,19 +404,19 @@ export const guides = [
             title: "Okta Sign-In Widget upgrade",
             subLinks: [
               {
-                title: "Upgrade your widget",
+                title: "Upgrade your Widget",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/const/navbar.const.js:407]: we don't capitalize when referring to widget any longer. That's a more recent change - you may have missed the discussion. 

```suggestion
                title: "Upgrade your widget",
```

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js [hunk 2]
```diff
@@ -404,19 +404,19 @@ export const guides = [
...
                 guideName: "oie-upgrade-sign-in-widget",
               },
               {
-                title: "Deprecated JavaScript methods in the widget",
+                title: "Deprecated Widget JS methods",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/const/navbar.const.js:411]:
```suggestion
                title: "Deprecated widget JS methods",
```

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js [hunk 3]
```diff
@@ -404,19 +404,19 @@ export const guides = [
...
                 guideName: "oie-upgrade-sign-in-widget-deprecated-methods",
               },
               {
-                title: "Updates to widget styling",
+                title: "Updates to Widget styling",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/const/navbar.const.js:415]:
```suggestion
                title: "Updates to widget styling",
```

File: packages/@okta/vuepress-theme-prose/const/navbar.const.js [hunk 4]
```diff
@@ -404,19 +404,19 @@ export const guides = [
...
                 guideName: "oie-upgrade-sign-in-widget-styling",
               },
               {
-                title: "Updates to widget i18n properties",
+                title: "Updates to Widget i18n",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-theme-prose/const/navbar.const.js:419]:
```suggestion
                title: "Updates to widget i18n",
```

**Review by susanharper-okta — APPROVED** [review]
just the lowercase w comment - approving to not hold you up

**Comment by chrisdmills-okta** [comment]
Thx @susanharper-okta !

**Pull request merged**